src/sync/cache.rs

```diff
@@ -3037,6 +3061,70 @@ mod tests {
         assert_eq!(counters.value_dropped(), KEYS, "value_dropped");
     }
 
+    // Ignored by default. This test cannot run in parallel with other tests.
+    #[test]
+    #[ignore]
```

Comment:
Add a step to the CI to run this test with `-j 1`.

Reply:
`-j 1` did not work well, so I added separate job steps for them via https://github.com/moka-rs/moka/pull/165/commits/4ac7d8c45dd4dcc47c3edcbb640ea0cf8d9930a1.

```console
> Run tests (debug, sync feature, thread-pool test for sync::Cache)

/home/runner/.cargo/bin/cargo test --lib --features sync sync::cache::tests::enabling_and_disabling_thread_pools -- --exact --ignored
    Finished test [unoptimized + debuginfo] target(s) in 0.10s
     Running unittests src/lib.rs (target/debug/deps/moka-5d8e30bdcb59a024)

running 1 test
test sync::cache::tests::enabling_and_disabling_thread_pools ... ok

test result: ok. 1 passed; 0 failed; 0 ignored; 0 measured; 85 filtered out; finished in 0.01s

> Run tests (debug, sync feature, thread-pool test for sync::SegmentCache)

/home/runner/.cargo/bin/cargo test --lib --features sync sync::segment::tests::enabling_and_disabling_thread_pools -- --exact --ignored
    Finished test [unoptimized + debuginfo] target(s) in 0.09s
     Running unittests src/lib.rs (target/debug/deps/moka-5d8e30bdcb59a024)

running 1 test
test sync::segment::tests::enabling_and_disabling_thread_pools ... ok

test result: ok. 1 passed; 0 failed; 0 ignored; 0 measured; 85 filtered out; finished in 0.02s
```